lib/ui/order/order_item_card.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class OrderItemCardState extends State<OrderItemCard> {
+  var _products = [];
+  List<dynamic> get products => _products;
```

Comment:
Why not have a single variable?

Reply:
This are private and public variables. What is used within the page is private and exposing it via a public variable.

Reply:
Any downsides to maintaining just one?